run_one_experiment.py

```diff
@@ -43,11 +43,20 @@
 # WARN: Avoid large NUM_SAMPLES in highly parallelized local experiment.
 # NUM_SAMPLES controls the number of LLM responses per query, which may exceed
 # your LLM's limit on query-per-second.
-NUM_SAMPLES = 2
+NUM_SAMPLES = 1
 MAX_TOKENS: int = 4096
 RUN_TIMEOUT: int = 30
 TEMPERATURE: float = 0.4
 
+# Query Strategy Parameters: Number of repeated queries for one prompt.
+# - [1]* found repeat is an effective strategy to improve the performance of
+#   LLM since LLM-based generation is a search process with randomness.
+# - Requery ensures repeat-k results instead of top-k results for all LLMs.
+# - A suggested usage is only set NUM_REPEATS while keep NUM_SAMPLES as 1.
+#
+# Link of [1]* will be updated later
+NUM_REPEATS = 6
```

Comment:
Would it be better to use a smaller default value? Just in case the user does not have much resource : )
We can add a comment recommending a better number, I recall you mentioned that this method shows its effectiveness with a large `NUM_REPEATS`.

They can specify a higher value via the arg you added above.